tools/replay/main.cc

```diff
@@ -16,6 +16,7 @@ int main(int argc, char *argv[]) {
       {"qcam", REPLAY_FLAG_QCAMERA, "load qcamera"},
       {"no-hw-decoder", REPLAY_FLAG_NO_HW_DECODER, "disable HW video decoding"},
       {"no-vipc", REPLAY_FLAG_NO_VIPC, "do not output video"},
+      {"ui-debug", REPLAY_FLAG_UI_DEBUG, "do output uiDebug and userFlag messages. may causes issues when used along with UI"}
```

Comment:
I think this use case can just use the `--allow` flag and they're blocked by default

Reply:
you're right, that's basically the same thing

Reply:
but if you want to replay everything, then you need to specify every service in `--allow`

Reply:
that's true, but this should still be a bit more generic. maybe make this flag something like `--all`?

Reply:
you're right, that seems nice